modules/phase_field/src/utils/Euler2RGB.C

```diff
@@ -289,28 +289,36 @@ euler2RGB(unsigned int sd, Real phi1, Real PHI, Real phi2, unsigned int phase, u
       // Increment to next symmetry operator if not in SST
       else
         index++;
+
+      // Error check if no solution found
+      if (index == nsym)
+        mooseError("Euler2RGB failed to map the supplied Euler angle into the SST!");
```

Comment:
Do we want this to fail hard? It seems to me that for an output capability we should soft fail (color the output in a specific way for failed matches).

Reply:
That is a good question. In an indirect way, the current code already soft fails by taking the solution from the last given symmetry operator and continuing along with incorrect results.  This is why the negative values made their way into the square root calculation. If the code is written correctly, we should never get a hard fail since a given set of valid Euler angles will always map into the standard stereographic triangle for a given crystal system. If they don't, I would rather know it so it can be fixed.  If you prefer to not have the hard fail, we could add a warning along with a specific output color.  Let me know what direction you want to go.

Reply:
I leave that up to you. Things that "should never happen" are usually covered by an assertion